0001-Enable-pbuffer-to-fix-most-deqp-pbuffer-testcases.patch

```diff
@@ -0,0 +1,291 @@
+From 882bd46068aec12aadfbdec2c25b92f118ca9aad Mon Sep 17 00:00:00 2001
+From: Liu Zhiquan <zhiquan.liu@intel.com>
+Date: Mon, 7 Nov 2016 15:32:17 +0800
+Subject: [PATCH] Enable pbuffer to fix most deqp pbuffer testcases.
+
+TODO: support EGL_SWAP_BEHAVIOR_PRESERVED_BIT for
+window surface.
+
+Bug ID: https://github.com/android-ia/device-androidia/issues/6
```

Comment:
We want to send it out to Mesa mailing list for review. You dont want to have Bug ID and Test in your commit message. Randy can help fine tune the commit message before sending it to Mesa mailing list. 

Include the Deqp tests which are expected to pass after this patch.